libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -43,40 +43,198 @@ export class LuisRecognizer extends IntentRecognizer {
             this.options = { appId: appId, subscriptionKey: subscriptionKey as string };
         } else {
             this.options = Object.assign({}, appId);
+        }   
+
+        // We need to set verbose to true in order to get all intents
+        if(this.options.options) {
+            this.options.options.verbose = true;
```

Comment:
Is all the right default?  Where do you use the explicit options?

Reply:
I'm kind of taking that control from the developer as I explained in my other comment, in favor of a consistent response, but happy to give the control back to the developer.

Reply:
I'll give the control back to the developer for the time being.